### PR TITLE
Add Gearman G2 chart

### PR DESCRIPTION
**Pull request by tamalsaha**
[G2](https://github.com/appscode/g2) is a modern implementation of Gearman server in GO.

### CODE REVIEW

File: stable/g2/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "name" . }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  replicas: {{.Values.replicaCount}}
+  template:
+    metadata:
+      labels:
```
Comment (unguiculus) [inline stable/g2/templates/deployment.yaml]: `chart` and `heritage` labels should be removed here.

File: stable/g2/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "name" . }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  replicas: {{.Values.replicaCount}}
```
Comment (unguiculus) [inline stable/g2/templates/deployment.yaml]: Add space after `{{` and before `}}` and make sure it's consistent throughout the chart.

File: stable/g2/templates/rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  verbs: ["get", "create", "update"]
+- apiGroups: [""]
+  resources:
+  - services
+  verbs: ["create", "update"]
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["list", "create", "update", "delete"]
+---
```
Comment (unguiculus) [inline stable/g2/templates/rbac.yaml]: Please create separate file.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
I have addressed your comments.

**Comment by unguiculus** [comment]
Actually, I wanted you to fix the whitespace issue around braces in all files. Sorry, if this was not clear.

**Comment by tamalsaha** [comment]
Should be fixed now.  It seems that the vscode extensions linting doed not identify this issue. PTAL.

File: stable/g2/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+$ kubectl api-versions | grep rbac
+```
+
+You also need to have the following parameter on the api server. See the following document for how to enable [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
+
+```
+--authorization-mode=RBAC
+```
+
+If the output contains "beta" or both "alpha" and "beta" you can may install with enabling the creating of rbac resources (see below).
```
Comment (unguiculus) [inline stable/g2/README.md]: Incorrect grammar. Try to improve the wording.

File: stable/g2/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (unguiculus) [inline stable/g2/templates/_helpers.tpl:1]: This is an outdated version of the file with wrong truncation. Update it from this example: https://github.com/kubernetes/helm/blob/master/docs/examples/nginx/templates/_helpers.tpl

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
Updated. PTAL.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**